exporter/awsemfexporter/emf_exporter.go

```diff
@@ -107,6 +107,12 @@ func (emf *emfExporter) pushMetricsData(_ context.Context, md pmetric.Metrics) e
 	for _, groupedMetric := range groupedMetrics {
 		cWMetric := translateGroupedMetricToCWMetric(groupedMetric, emf.config)
 		putLogEvent, err := translateCWMetricToEMF(cWMetric, emf.config)
+		if emf.config.EnhancedContainerInsights && (putLogEvent != nil &&
+			putLogEvent.InputLogEvent != nil &&
+			putLogEvent.InputLogEvent.Message != nil) && *putLogEvent.InputLogEvent.Message == "" {
+			emf.config.logger.Info("Dropping Prometheus job instance metrics log for EnhancedContainerInsights")
```

Comment:
is this info message too specific?  This does not seem to do anything prometheus specific, it looks like it just drops inputs where the message is ""

--

Should this be debug? IIRC the other `dropping` log line is a debug log.

Reply:
Updated the log line to debug